langs/swift/Dockerfile

```diff
@@ -1,28 +1,72 @@
+  | tar xz --strip-components 1
 
 RUN strip /usr/bin/swift /usr/bin/swift-frontend \
- && rm -r /usr/lib/swift/embedded                \
  && find /usr/lib/swift -name '*.so' -exec strip {} +
 
+RUN rm -r /usr/lib/swift/linux/x86_64
+
 FROM codegolf/lang-base
 
-COPY --from=0 /lib/x86_64-linux-gnu                     /lib/x86_64-linux-gnu
-COPY --from=0 /lib64                                    /lib64
-COPY --from=0 /usr/bin/swift /usr/bin/swift-frontend    /usr/bin/
-COPY --from=0 /usr/include                              /usr/include
-COPY --from=0 /usr/lib/clang/17/include                 /usr/lib/clang/17/include
-COPY --from=0 /usr/lib/swift                            /usr/lib/swift
-COPY --from=0 /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 \
-              /usr/lib/x86_64-linux-gnu/libstdc++.so.6  \
-              /usr/lib/x86_64-linux-gnu/libuuid.so.1    /usr/lib/x86_64-linux-gnu/
+COPY --from=0 /lib/x86_64-linux-gnu/libc.so.6                       \
+              /lib/x86_64-linux-gnu/libdl.so.2                      \
+              /lib/x86_64-linux-gnu/libgcc_s.so.1                   \
+              /lib/x86_64-linux-gnu/libm.so.6                       \
+              /lib/x86_64-linux-gnu/libncurses.so.6                 \
+              /lib/x86_64-linux-gnu/libpthread.so.0                 \
+              /lib/x86_64-linux-gnu/librt.so.1                      \
+              /lib/x86_64-linux-gnu/libtinfo.so.6                   \
+              /lib/x86_64-linux-gnu/libutil.so.1                    \
+              /lib/x86_64-linux-gnu/libz.so.1                       /lib/
+COPY --from=0 /lib64/ld-linux-x86-64.so.2                           /lib64/
+COPY --from=0 /usr/bin/swift /usr/bin/swift-frontend                /usr/bin/
+COPY --from=0 /usr/include                                          /usr/include
+COPY --from=0 /usr/lib/x86_64-linux-gnu/libsqlite3.so.0             \
+              /usr/lib/x86_64-linux-gnu/libstdc++.so.6              \
+              /usr/lib/x86_64-linux-gnu/libuuid.so.1                /usr/lib/
+COPY --from=0 /usr/lib/swift/host/*BasicFormat.so                   \
+              /usr/lib/swift/host/*CompilerPluginMessageHandling.so \
+              /usr/lib/swift/host/*Diagnostics.so                   \
+              /usr/lib/swift/host/*IDEUtils.so                      \
+              /usr/lib/swift/host/*Operators.so                     \
+              /usr/lib/swift/host/*Parser.so                        \
+              /usr/lib/swift/host/*ParserDiagnostics.so             \
+              /usr/lib/swift/host/*Syntax.so                        \
+              /usr/lib/swift/host/*SyntaxBuilder.so                 \
+              /usr/lib/swift/host/*SyntaxMacroExpansion.so          \
+              /usr/lib/swift/host/*SyntaxMacros.so                  /usr/lib/swift/host/
+COPY --from=0 /usr/lib/swift/linux                                  /usr/lib/swift/linux
```

Comment:
Perhaps a bit more post addressing this final directory, but that'll happen some other time. At least there's line 15 already.